lua-to-luau (#289)
Roblox/creator-docs: content/en-us/tutorials/scripting/basic-scripting/intro-to-scripting.md

## Inserting a Script

> Code in Roblox is written in a language called **[Luau](../../../luau/index.md)**, and it's stored and ran from scripts. You can put scripts in various containers in the **Explorer**. If you put a script under a **Part**, Roblox will run the code in the script when the part is loaded into the game.

Reviewer: ```suggestion
Code in Roblox is written in a language called [Luau](../../../luau/index.md) which you can put in scripts within various containers in the **Explorer**. If you put a script under a **Part**, Roblox will run the script's code when the part is loaded into the game.
```
